pkg/model/components/etcdmanager/model.go

```diff
@@ -238,7 +243,43 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster kops.EtcdClusterSpec, instance
 		} else {
 			pod = podObject
 		}
+	}
+
+	{
+		for _, etcdVersion := range etcdSupportedVersions() {
+			initContainer := v1.Container{
+				Name:    "init-etcd-" + strings.ReplaceAll(etcdVersion, ".", "-"),
+				Image:   etcdSupportedImages[etcdVersion],
+				Command: []string{"/bin/sh"},
+				Args: []string{
+					"-c",
+				},
+				VolumeMounts: []v1.VolumeMount{
+					{
+						MountPath: "/opt",
+						Name:      "bin",
+					},
+				},
+			}
+			// Add the command for copying the etcd binaries
+			var command string
+			if semver.MustParse(etcdVersion).GE(semver.MustParse("3.4.13")) {
+				command = "cp /usr/local/bin/etcd /opt/etcd-v" + etcdVersion + "/etcd && cp /usr/local/bin/etcdctl /opt/etcd-v" + etcdVersion + "/etcdctl"
```

Comment:
Nit: I'm guessing we somehow don't, but we don't need the mkdir here?  I'm also confused because this looks like it is doing the same thing as the "other" command

Reply:
Newer images bundle a custom go based `cp` utility that recursively creates the necessary dirs.
https://github.com/kubernetes/kubernetes/pull/91171

Reply:
Longer term we should take the shell out of the container and use a custom Go binary for this.